README.md

```diff
@@ -75,19 +101,24 @@ func NewConfig() *Config
 
 #### New
 
-`New` creates an instance of your module. It must return an `io.Closer` and an `error.
+`New` creates an instance of your module. It must return an `io.Closer` and an `error`.
 The function takes a `context.Context`, the configuration structure returned by `NewConfig`,
 `Info` and `UI` objects. `Info` and `UI` are located in `github.com/glasslabs/looking-glass/module/types`.
 
 ```go
-func New(_ context.Context, cfg *Config, info types.Info, ui types.UI) (io.Closer, error)
+func New(ctx context.Context, cfg *Config, info types.Info, ui types.UI) (io.Closer, error)
 ```
 
+#### Dependencies
+
+All dependencies must vendored except for `github.com/glasslabs/looking-glass/module/types`. 
```

Comment:
I'd suggest explaining in more details what it means for a dependency to be vendored in Go, or at least to add a link over `vendored` that points to something official talking about it.